pandas/tests/frame/methods/test_replace.py

```diff
@@ -1403,3 +1403,17 @@ def test_replace_with_duplicate_columns(self, replacement):
         result["B"] = result["B"].replace(7, replacement)
 
         tm.assert_frame_equal(result, expected)
+
+    def test_replace_period_ignore_float(self):
+        """
+        Regression test for GH#34871: if df.replace(1.0, 0.0) is called on a df
+        with a Period column the old, faulty behavior is to raise TypeError.
+        """
+        df = pd.DataFrame({"Per": [pd.Period("2020-01")] * 3})
+        df_after_replace = df.replace(1.0, 0.0)
+
+        df_expected_result = pd.DataFrame({"Per": [pd.Period("2020-01")] * 3})
```

Comment:
can you call things result= and expected= its much easier to read (and doesn't mix in the same name)

Reply:
done!